clients/python/girder_client/__init__.py

```diff
@@ -874,6 +874,57 @@ def uploadFileToItem(self, itemId, filepath, reference=None, mimeType=None, file
         with open(filepath, 'rb') as f:
             return self._uploadContents(obj, f, filesize, progressCallback=progressCallback)
 
+    def uploadFileToFolder(self, folderId, filepath, reference=None, mimeType=None, filename=None,
```

Comment:
@manthey I'm late to the party, but is there anything precluding `filepath` from being a file-like object similar to other methods in girder-client? I'm dealing with in-memory bytes that I want to upload to Girder as part of a Celery task, so it would be nice to avoid writing it to disk first.